
<launch>
<node pkg="turtlebot_deployment" type="ekf" name="ekf_filter" output="screen">
<param name="robot_name"	value="$(arg robot)"/>
<param name="sendAll"	value="$(arg send)"/>
</node>
</launch>
